package.json

```diff
@@ -225,22 +225,30 @@
     "cover-integration": "cross-env COVERAGE=true nyc --reporter=lcovonly yarn mocha \"test/integration/**/*.test.[tj]s\" && node -e \"require('fs').renameSync('coverage/lcov.info', 'coverage/integration.info')\"",
     "cover-unit": "cross-env COVERAGE=true nyc --reporter=lcovonly yarn mocha \"test/unit/**/*.test.[tj]s\" && node -e \"require('fs').renameSync('coverage/lcov.info', 'coverage/unit.info')\"",
     "merge-coverage": "lcov-result-merger \"coverage/*.info\" \"coverage/lcov.info\"",
-    "----------------------------------------- local test dbs ------------------------------------------": "",
+    "----------------------------------------- bulk tests ------------------------------------------": "",
+    "start-all": "grep 'bash dev' package.json | grep start | grep -vP \"(all)\" | cut -d'\"' -f4 | sort | xargs -n1 | grep dev | xargs -n1 bash",
+    "test-all": "grep 'test-' package.json | grep -vP \"(unit|integration|smoke|typings)\" | cut -d'\"' -f4 | xargs -n4 yarn ",
```

Comment:
ibmi smoke tests should not be part of this, they take ages to run and require private credentials.

If typings are included, then `lint` could be included too